features/step_definitions/authentication_steps.rb

```diff
@@ -0,0 +1,27 @@
+Given(/^I have an account on the site$/) do
+  FactoryGirl.create(:user, email: "user@example.com", password: "password")
+end
+
+Given(/^My account is confirmed$/) do
+  User.last.confirm
```

Comment:
my only comment is that i think "convention" would have this set up so you could call it as `User.last.is_confirmed?` or `User.last.confirmed?`, but that's just a stylistic opinion - no need to change it